extensions/3rdparty/LyricWiki/Tag_Lyric.php

```diff
@@ -92,7 +92,7 @@ function lyricTagCss($out)
 	return true;
 }
 
-function renderLyricTag($input, $argv, $parser)
+function renderLyricTag( $input, array $argv, Parser $parser, PPFrame $frame )
```

Comment:
Since the method-signature has to match the convention for the parser-hook... I'll leave it as is.